packages/nc-gui-v2/composables/useViewSorts.ts

```diff
@@ -68,11 +54,16 @@ export function useViewSorts(
   }
 
   const deleteSort = async (sort: SortType, i: number) => {
-    if (isUIAllowed('sortSync') && sort.id && !isPublic.value && !isSharedBase) {
+    if (isUIAllowed('sortSync') && sort.id && !isPublic.value && !isSharedBase.value) {
       await $api.dbTableSort.delete(sort.id)
```

Comment:
add a try catch for api call. throw errors using `message.error(await extractSdkResponseErrorMsg(e))`